open_city_profile/settings.py

```diff
@@ -214,3 +216,7 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Ilmoitin settings
+
+MAILER_EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```

Comment:
should probably use env variables. [example from venepaikka](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/berth_reservations/settings.py#L35)

Reply:
also those mailer variables should probably be included as well. I think we will still use mailgun, [again venepaikka as example](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/berth_reservations/settings.py#L67)

Reply:
Will do!